spotbugs-tests/src/test/java/edu/umd/cs/findbugs/ba/Issue1367Test.java

```diff
@@ -15,12 +19,12 @@
 /**
  * @see <a href="https://github.com/spotbugs/spotbugs/issues/1367">The related GitHub issue</a>
  */
-public class Issue1367Test {
-    @Rule
-    public SpotBugsRule spotbugs = new SpotBugsRule();
+@ExtendWith(SpotBugsExtension.class)
+class Issue1367Test {
 
     @Test
-    public void test() {
+    @DisabledOnJre(JRE.JAVA_11)
```

Comment:
Are you sure that this test needs to be disabled on Java 11?

Reply:
Yes, it only ever was for jdk > 11, see line 28, its specifically aligned to our usage of jdk 17.